tests/bpftool_tests/bpftool_tests.cpp

```diff
@@ -124,41 +124,44 @@ TEST_CASE("prog load map_in_map", "[prog][load]")
     REQUIRE(result == 0);
 }
 
-TEST_CASE("prog attach by interface alias", "[prog][load]")
-{
-    int result;
-    std::string output;
-    char command[80];
-    sprintf_s(
-        command, sizeof(command), "bpftool --legacy prog load droppacket%s droppacket", EBPF_PROGRAM_FILE_EXTENSION);
-
-    output = run_command(command, &result);
-    REQUIRE(output == "");
-    REQUIRE(result == 0);
-
-    output = run_command("bpftool prog show", &result);
-    REQUIRE(result == 0);
-    std::string id = std::to_string(atoi(output.c_str()));
-    size_t offset = output.find(" map_ids ");
-    REQUIRE(offset != std::string::npos);
-    std::string map_id1 = std::to_string(atoi(output.substr(offset + 9).c_str()));
-    offset = output.find(",");
-    REQUIRE(offset != std::string::npos);
-    std::string map_id2 = std::to_string(atoi(output.substr(offset + 1).c_str()));
-    REQUIRE(output == id + ": xdp_test  name DropPacket  \n  map_ids " + map_id1 + "," + map_id2 + "\n");
-
-    // Try attaching to an interface by friendly name.
-    output = run_command(("bpftool net attach xdp id " + id + " dev \"Loopback Pseudo-Interface 1\"").c_str(), &result);
-    REQUIRE(result == 0);
-
-    output = run_command(("netsh ebpf delete prog " + id).c_str(), &result);
-    REQUIRE(output == "\nUnpinned " + id + " from droppacket\n");
-    REQUIRE(result == 0);
-
-    output = run_command("bpftool prog show", &result);
-    REQUIRE(output == "");
-    REQUIRE(result == 0);
-}
+// Once the libbpf APIs are implemented in the xdp-for-windows repo, these tests should be migrated.
```

Comment:
I'd suggest an #ifdef rather than commenting out code.  It's easier on github diffs.

Reply:
Switched to '#if 0' - let me know if that looks ok